ROADMAP.md

```diff
@@ -0,0 +1,60 @@
+# Katib 2019 Roadmap
+
+This document provides a high level view of where Katib will likely grow in 2019.
+
+Katib's Critical User Journey (CUJ) can be found [here](https://bit.ly/2QNKMwt).
+
+# Katib 1.0 Readiness
+
+* Stabilize APIs for StudyJobs
+	* Beta by end of Q2, 1.0 by end of Q4
+	* Formalize naming conventions (we use different names like katib vs vizier in different places)
+* Fully integrate katib with existing E2E examples:
+	* Xgboost
+	* Mnist
+	* GitHub issue summarization
+* Publish API documentation, best practices, tutorials
+* [Issues list](https://github.com/kubeflow/katib/issues)
+
+
+# Enhance HP Tuning Experience
+
+The objectives here are organized around the three stages defined in the CUJ:
+
+## 1. Defining Model and Parameters
+
+Integration with KF distributed training components
+* TFJob
+* PyTorch
+* Allow Katib to support other operator types generically
```

Comment:
Is there any plan/discussion/issue on this? It might be good to test this with mpi-operator.

Reply:
There are some discussions in these issues:
https://github.com/kubeflow/katib/issues/317
https://github.com/kubeflow/katib/issues/341